hack/build.sh

```diff
@@ -106,8 +106,8 @@ function build_caa_payload_arch_specific() {
 		--build-arg YQ_CHECKSUM="${YQ_CHECKSUM}" \
 		-f Dockerfile \
 		${tag_string} \
-		--push \
```

Comment:
here maybe we need `--load` to replace `--push`

Reply:
Added --load